### ISSUE TITLE
Question about logged-in user on DB

### ISSUE THREAD

**Issue by 9ummy** [swsnu/swppfall2021#67]
After a user logs in and tries to write an article, logged-in user information is needed in article create page. Since I thought there could be more than one logged-in user on DB when the backend side is ready, I thought it was not enough to specify the logged-in user only with `logged_in` property in DB, and used `sessionStorage` to store logged-in user information.

Is it okay to use `sessionStorage` to store login user information?
Or should I suppose that there is always one logged-in user on DB and specify logged-in user with `logged_in` property?

Thanks in advance!

**Comment by kdh0102** [comment]
> Or should I suppose that there is always one logged-in user on DB and specify logged-in user with logged_in property?

Yes, it is enough to assume there is only one logged-in user on DB in this homework.

**Comment by 9ummy** [comment]
Thanks a lot 👍
